examples/jsm/renderers/CSS3DRenderer.js

```diff
@@ -105,12 +105,16 @@ class CSS3DRenderer {
 
 		this.domElement = domElement;
 
+		const viewElement = document.createElement( 'div' );
+		viewElement.style.transformOrigin = '0 0';
+		viewElement.style.pointerEvents = 'none';
+		domElement.appendChild( viewElement );
```

Comment:
As I mentioned https://github.com/mrdoob/three.js/issues/21723#issuecomment-839426359
a wrapper element is required to handle camera-view.